### ISSUE TITLE
Is it possible to get Ratcliff-Obershelp similarity implemented?

### ISSUE THREAD

**Issue by luka-TU** [smashedtoatoms/the_fuzz#9]
Hi!

Is this project still active?

**Comment by smashedtoatoms** [comment]
Yep.  I don't use it much, but I work on it occasionally.  I can look into that algorithm and see how tricky adding will be.  You're welcome to take a stab at it too.  A lot of the algorithms have been done by people other than me.
